### PR TITLE
feat(create the docs): add docz to create the documentation

### PR DESCRIPTION
**Pull request by nicholasess**
Hi Everyone!

I created the documentation of react-native-share with [docz](https://www.docz.site/).

To see the example run in netlify -> [link](https://react-native-share.netlify.com/)

Now, I need the suggestion to README.md and you create a domain to host doc.

- [ ] README.md
- [ ] Github Pages
- [x] Images worked
- [x] Dev dependencies

Thanks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,5 +61,11 @@
     "release:minor": "node ./changelog --minor && npm version minor && git push origin && git push origin --follow-tags",
     "release:patch": "node ./changelog --patch && npm version patch && git push origin && git push origin --follow-tags",
     "release:version": "node ./changelog --version $VERSION && git push origin && git push origin --follow-tags"
+  },
+  "dependencies": {
```
Comment (jgcmarins) [inline package.json]: Can we keep `docz` as a dev dependency?

**Comment by julioxavierr** [comment]
Nice!!!

**Comment by jgcmarins** [comment]
I am very impressed on how easy you turned a README file into a beautiful and awesome documentation.

**Comment by gtkatakura-bysoft** [comment]
Awesome job!

Hey, images are broken in the example, see:

captura de tela 2018-08-31 as 10 05 51

**Comment by jgcmarins** [comment]
Nice! I think we need a few changes yet.
CI is failing, but I think it's not related to this PR.
I'll fix it on `master`.
```
#!/bin/bash -eo pipefail
yarn install
yarn install v0.24.6
[1/4] Resolving packages...
[2/4] Fetching packages...
error koa@2.5.2: The engine "node" is incompatible with this module. Expected version "^4.8.4 || ^6.10.1 || ^7.10.1 || >= 8.1.4".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Exited with code 1
```
screen shot 2018-09-03 at 09 52 46

**Comment by jgcmarins** [comment]
don't worry, it is not related to this PR

**Pull request closed without merging**